api/main.py

```diff
@@ -141,6 +141,7 @@ async def authorize_user(node_id: str, user: User = Depends(get_current_user)):
           response_model_by_alias=False)
 async def post_user(
         username: str, password: Password,
+        email: str,
```

Comment:
Just for readability:
```suggestion
        username: str, password: Password, email: str,
```